assets/styles/homepage.scss

```diff
@@ -0,0 +1,220 @@
+// Imports -------------- //
+
+@import "../../node_modules/uswds/src/stylesheets/all";
```

Comment:
We need to add the node package to our `package.json` file in order for this to work correctly. Spoke with @rogeruiz too and this can definitely be added by pointing to here for now:

`git+https://github.com/18F/web-design-standards.git#18f-pages-staging`

Sounds like we are close to completing the the USWDS node package too and can update that link once it's GTG.